docker/regtest/docker-compose.yml

```
@@ -1,6 +1,14 @@
 version: "3"
 
 services:
+  nginx:
+    image: nginx
```

Currently the container will be named `regtest_nginx_1` - all others start with `jm_regtest`. Should we add it here too and maybe provide a meaningful description like "test_basepath"?

```suggestion
    container_name: jm_regtest_nginx_test_basepath
    image: nginx
```

Just a nit. Not important.